### DISCUSSION TITLE
About ownership of the Formulae/Casks

### DISCUSSION THREAD

**Discussion by weyderfs** [Homebrew/discussions#4125] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
No data
```

### Output of `brew doctor`

```shell
No data
```

### Description of issue

Guys,
How works when the project of a specific HB Formula it's "deprecated" or the maintainer no long gives more support to the project original? Is there some way to change the ownership? 

I'm talking about this HB Formula: [tgenv](https://formulae.brew.sh/formula/tgenv#default).

There's the following situation. The original project `tfenv` Cunymatthieu looks like abandoned the last update it was in the 2019 thenceforth, some issues or pull request are open without review and the user Cunymatthieu dont't reply the question about the project.

The original repository: [cunymatthieu/tgenv](https://github.com/cunymatthieu/tgenv)

So, me and others community users proposed keep to maintainer the project and starts a new project [tgenv/tgenv](https://github.com/tgenv/tgenv).

**Comment by SMillerDev** [comment]
If upstream confirms the fork should be considered the new source of truth we can change the formula source. Additionally, if enough other packagers are convinced the source should be switched the formula source can be changed too.

**Reply by iokiwi** [reply]
Hi @SMillerDev, I am revisiting this one almost 18 months later. Our problem is that the sole maintainer of upstream has not updated the repository, addressed any issues or pull requests to their project in the three years prior to us initiating this conversation or the 18 months that have elapsed since.

I am making new attempts to get in contact with the maintainer via channels outside of Github incase they simply don't have access to that GitHub account or the email address it is registered to.

If we still don't get a response from the maintainer, what are our options then? It seem to be a cut and dry case of project abandonment.

Does our suggestion of publishing our fork under `tgenv/tgenv` work or does that clash with the `tgenv` name space?

**Comment by iokiwi** [comment, marked as answer]
@weyderfs this could be a hard sell for the homebrew team. Handing over a control of a namespace is a pretty big deal and has pretty significant ramifications.

In the meantime we could publish to homebrew under tgenv/tgenv. 

If we ever get tractions with upstream or the home brew team we can just ocupy both namespaces

**Reply by weyderfs** [reply]
@iokiwi great, to me ok, in the future uses the brew formulae tgenv/tgenv.

**Comment by iokiwi** [comment]
Update: The tgenv formula has been turned over to our fork. We consider this matter resolved
https://github.com/tgenv/tgenv/issues/15
